src/python/grapl-template-generator/grapl_template_generator/grapl-templates/grapl-python-service-template/{{cookiecutter.project_slug}}/.gitignore

```diff
@@ -0,0 +1,131 @@
+# Created by https://www.gitignore.io/api/python
+# Edit at https://www.gitignore.io/?templates=python
```

Comment:
(we probably don't need this big giant gitignore, but it doesn't hurt anything. there is precedent for `src/lang/project/.gitignore` files.)